curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-algorithmic-thinking-by-building-a-number-sorter/6410dfb965c72108196ef24a.md

```diff
@@ -0,0 +1,362 @@
+---
+id: 6410dfb965c72108196ef24a
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+Because you need to compare elements, you'll need to use a nested `for` loop. This loop should iterate through every element in the array *except* the last one. Use `j` for your inner loop's iterator variable.
```

Comment:
```suggestion
Because you need to compare elements, you'll need to use a nested `for` loop. This loop should iterate through every element in the array *except* the last one. Use `j` as your inner loop's iterator variable.
```